lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,141 @@
+    type: CodeableConcept
+    description: The type for the Organization.
+    binding:
+      description: Indicates the type of Organization.
+      terminology:
+        - display: OrganizationType
+          system: http://hl7.org/fhir/ValueSet/organization-type
+          info_link: http://terminology.hl7.org/CodeSystem/organization-type.html
+        - display: Millennium Organization Identifier (Alias) Types
+          system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/278
+          info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-278-organization-type
+
+  - name: resourceType
+    required: 'Yes'
+    cardinality: 1..1
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Organization"
+      }
+    note: resourceType must be Organization.
+    url: https://hl7.org/fhir/R4/json.html#resources
+
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: The identifiers for the Organization.
+    action: create
+    example: |
+      {
+        "identifier": [
+                {
+                    "system": "http://hl7.org/fhir/sid/us-npi",
+                    "value": "FHIR-NPI-10191",
+                    "type": {
+                        "coding": [
+                            {
+                                "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                                "code": "NPI"
+                            }
+                        ]
+                    }
+                }
+            ]
+      }
+    note: |
+      <ul>
+        <li> If set, The provided identifier must meet the following conditions:
+          <ul>
+            <li> Must contain the `system`, `value`, and `type` fields.</li>
+            <li> Must contain only an NPI, OID or TAXID system. </li>
+          </ul>
+        </li>
+      </ul>
+    url: http://hl7.org/fhir/R4/datatypes.html#Identifier
+
+  - name: active
+    required: 'No'
+    cardinality: 0..1
+    type: boolean
+    description: Whether this patient record is in active use.
+    action: create
+    example: |
+      {
+        "active": true
+      }
+    note: If set, value must be true.
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.active
+
+  - name: name
+    required: 'Yes'
+    cardinality: 0..1
+    type: OrganizationName
+    description: The names associated with the individual.
+    action: create
+    example: |
+      {
+        "name": "example-organization"
+      }
+    note: |
+      <ul>
+        <li> At least one name <b>must</b> be provided. </li>
+      </ul>
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.name
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: OrganizationType
+    description: The Organization Type.
+    action: create
+    example: |
+      {
```

Comment:
spacing is off

Reply:
fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R31

Reply:
Pretty sure the spacing is still off

Reply:
fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/f2f4341b4d5d18c180e40f44cd8daa2c3e3ed8fd

Reply:
Fixed: https://github.com/cerner/fhir.cerner.com/pull/326/commits/fb4db54a3c8e0b86d489a4e434f5277a56342255#diff-b08823bfbc91ea717475d1d2372baf64R64